src/routes/members/gigs/[gig_id([0-9]+)]/edit.svelte

```diff
@@ -0,0 +1,1133 @@
+    background: var(--background);
+    left: 50%;
+    transform: translateX(-50%);
+    z-index: 10;
+    border-radius: 5px;
+    font-size: 1.2rem;
+    width: 150px;
+    box-shadow: 0 0 4px var(--form_color);
+  }
+
+  .gig-contact {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    padding-left: 0.25em;
+  }
+
+  .gig-contact:nth-child(2n) {
+    background: rgba(var(--accent_triple), 0.1);
+  }
+
+  .button-group {
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+  }
+
+  .button-group button {
+    width: unset;
+    margin: 0.25em 0.25em;
+  }
+
+  hr {
+    margin: 1em 0;
+  }
+
+  *:disabled {
+    filter: opacity(0.5);
+  }
+
+  .gig-gig_cancelled::before {
+    content: "Cancelled: ";
+  }
+
+  .gig-gig_enquiry::before {
+    content: "Enquiry: ";
+  }
+
+  .gig-kit::before {
+    content: "Kit hire: ";
+  }
+
+  button.gig-preview {
+    margin-bottom: 0.75em;
+  }
+
+  .warnings {
+    padding: 0;
+    * {
+      margin: 0.5em 0;
+      box-sizing: border-box;
+      @include themeify($themes) {
+        background: themed("warningBackground");
+        color: themed("warningColor");
+        border: 1px solid themed("warningBorderColor");
+      }
+      list-style-type: none;
+      padding: 0.15em 0.25em;
+    }
+  }
+
+  label.checkbox {
+    display: block;
+
+    input {
+      width: auto;
+    }
+  }
+</style>
+
+<svelte:head>
+  <title>{makeTitle(`Edit gig: ${title.trim()}`)}</title>
+</svelte:head>
+
+<svelte:window on:beforeunload="{unloadIfSaved}" on:keydown="{keyboardShortcuts}" />
+
+<h1>Gig editor</h1>
+<h2 class="gig-{typeCode}">
+  {title.trim()}
+  {#if !saved}&nbsp;(unsaved changes){/if}
+</h2>
+
+<a href="/members/gigs/{id}" rel="{!saved ? 'external' : undefined}">View gig summary</a><br /><br />
+
+{#if previewSummary}
+  <button class="gig-preview" on:click="{() => (previewSummary = false)}" data-test="gig-edit-{id}-hide-preview">Hide
+    gig preview</button>
+  <Summary gig="{summaryGig}" displayLinks="{false}" />
+  <!-- TODO show public advert when that's implemented (#35) -->
+{:else}
+  <button class="gig-preview" on:click="{() => (previewSummary = true)}" data-test="gig-edit-{id}-show-preview">Show gig
+    preview</button>
+{/if}
+
+<p>
+  {#if editing_time}
+    This gig was last edited at
+    {DateTime.fromISO(editing_time).toFormat('HH:mm dd/LL/yyyy')}
+    {#if editing_user}
+      &nbsp;by user
+      <a href="/members/users/{editing_user.id}">{editing_user.first}&#32;{editing_user.last}</a>
+    {/if}.
+  {/if}
+  {#if posting_time}
+    It was created at
+    {DateTime.fromISO(posting_time).toFormat('HH:mm dd/LL/yyyy')}
+    {#if posting_user}
+      &nbsp;by user
+      <a href="/members/users/{posting_user.id}">{posting_user.first}&#32;{posting_user.last}</a>
+    {/if}.
+  {/if}
+</p>
+
+<p>
+  Always remember to click 'Save' after changing anything to be safe! You should also be able to use Alt-S or
```

Comment:
is this still true? Can you make it slightly less annoying/vague? tell me one or maybe 2 ways to save not like 5, make auto-saving consistent or not at all?